### ISSUE TITLE
Proxy setup example

### ISSUE THREAD

**Issue by johngluckmdsol** [mdsol/mauth-client-ruby#9]
Would one of you be willing to set up the proxy and make it work with some external client (Like postman or paw).  We tried doing this but it appears there is a defect in the proxy.  Before I file a defect though, I just want to make sure I'm not misunderstanding how to operate the proxy.

Apparently this the java mauth-client actually works, but it would be nice to get the ruby one to work because I don't currently have a java setup.

**Comment by jfeltesse-mdsol** [comment]
Hey, seeing this issue just now sorry. It's been almost a year, is this still relevant?

**Comment by johngluckmdsol** [comment]
Yes, there is no documentation on how to get this actually work.  I struggled with it for an hour and gave up and then did so again.  I would love to see an actual working example
